backend/btrixcloud/basecrawls.py

```diff
@@ -177,11 +177,14 @@ async def get_crawl_out(
 
                 oid = res.get("oid")
                 if oid:
+                    origin = get_origin(headers)
                     res["pagesQueryUrl"] = (
-                        get_origin(headers)
-                        + f"/api/orgs/{oid}/crawls/{crawlid}/pagesSearch"
+                        origin + f"/api/orgs/{oid}/crawls/{crawlid}/pagesSearch"
                     )
 
+                    # this will now disable the downloadUrl in RWP
+                    res["downloadUrl"] = None
```

Comment:
```suggestion
                # this will now disable the downloadUrl in RWP
                res["downloadUrl"] = None
```
Not sure this needs to be conditional on oid.